zpay32/invoice_test.go

```diff
@@ -300,39 +300,9 @@ func TestDecodeEncode(t *testing.T) {
 				i.Destination = nil
 			},
 		},
-		{
```

Comment:
Why was this test case removed?

Reply:
When the spec was updated, a new invoice example had to be provided. Rusty provided an updated example (see https://github.com/lightningnetwork/lightning-rfc/pull/317/commits/c85c64c51e55bdc2a412050497e185417c9010db#diff-16621ac6a270af29f36f0a4f900955fcL339) for one of the tests, but I couldn't find one for this test case. I'm not familiar with how these new invoice examples are created though.